js/client/modules/@arangodb/inspector.js

```diff
@@ -613,12 +614,20 @@ function getServerData(arango) {
           tmp = executeExternalAndWait(
             '/bin/bash', ['-c', 'uname -a | tee /tmp/inspector-uname.out > /dev/null']);
           const uname = fs.readFileSync('/tmp/inspector-uname.out', 'utf8').slice(0,-1);
-          var top;
+          tmp = executeExternalAndWait(
+            '/bin/bash', ['-c', 'cat /etc/*-release | tee /tmp/inspector-release.txt > /dev/null']);
+          const release = fs.readFileSync('/tmp/inspector-release.txt', 'utf-8').slice(0,-1);
+          var top, du;
           if (status.pid !== undefined) {
             tmp = executeExternalAndWait(
               '/bin/bash', ['-c', 'top -b -H -p ' + status.pid + ' -n 1 | tee /tmp/inspector-top.out > /dev/null']);
             top = fs.readFileSync('/tmp/inspector-top.out', 'utf8').slice(0,-1);
           }
+          if (status.hasOwnProperty("datapath")) {
+            tmp = executeExternalAndWait(
+              '/bin/bash', ['-c', 'du -sh ' + status.datapath + ' | tee /tmp/inspector-du.out > /dev/null']);
```

Comment:
This is dangerous. What is "datapath" is "Hallo World How Are You" (with spaces)?

Reply:
good point